### PR TITLE
Add New Instructions for Using brew update-reset to CLI Docs

### PR DESCRIPTION
**Pull request by kwcanuck**
[Slack request](https://astronomer.slack.com/archives/C03CT3DNZMH/p1671028839193419).

### CODE REVIEW

**Comment by BasPH** [comment]
I'm a little hesitant about suggesting `brew update-reset`. Would only suggest `brew update`.

From https://medium.com/@samfirminger/odd-solution-to-homebrew-update-fail-homebrew-core-stopping-install-of-a-package-via-brew-f1064b786144:
> “The update-reset version of this command resets all formulae to be identical to the contents of their remote repositories, deleting any local changes. It is only used as a last-resort effort to fix a problem (it’s like unplugging Homebrew and plugging it back in).”

File: astro/cli/install-cli.md [hunk 1]
```diff
@@ -42,6 +42,25 @@ To install the latest version of the Astro CLI, run the following command:
 ```sh
 brew install astro
 ```
+If the latest version of the Astro CLI is not installed when you run this command:
+
+1. Run one of the following commands to update all package definitions (formulae) and Homebrew:
```
Comment (BasPH) [inline astro/cli/install-cli.md:47]:
```suggestion
1. Run the following command to update all package definitions (formulae) and Homebrew:
```

**Review by BasPH — APPROVED** [review]

**Review by iamakashladdha — COMMENTED** [review]
The changes look good to me. Thanks, @BasPH, and @kwcanuck

File: astro/cli/install-cli.md [hunk 2]
```diff
@@ -42,6 +42,25 @@ To install the latest version of the Astro CLI, run the following command:
 ```sh
 brew install astro
 ```
+If the latest version of the Astro CLI is not installed when you run this command:
```
Comment (jwitz) [inline astro/cli/install-cli.md:45]:
```suggestion

If the latest version of the Astro CLI is not installed when you run this command:
```

**Pull request merged**